js/script.js

```diff
@@ -1 +1,6 @@
-console.log ('hello');
\ No newline at end of file
+"use strict"
+let user = '',
```

Comment:
userName і colorGlas це властивості сутності user, тому краще було б використати об'єкт {}
Наприклад
```suggestion
const user = {
firstName: "Назар",
eyeColour: "Синій"
}
```